front/src/app/scenes/contract/contract.component.html

```diff
@@ -47,6 +47,21 @@
             <option *ngFor="let compiler of compilers">{{compiler}}</option>
           </select>
         </div>
+        <div class="form-group">
+          <label for="evm">EVM</label>
+          <select
+              class="form-control"
+              id="evm"
+              formControlName="evm_version">
```

Comment:
what it affects? or you haven't added a code yet?

Reply:
This should be working end-to-end. `evm_version` is passed along through the backend to the `solc` docker command.